.gitignore

```diff
@@ -187,3 +187,6 @@ dmypy.json
 # Visual Studio Code Files
 .vscode
 .devcontainer
+
+# Visual Studio Code Files
```

Comment:
This line is duplicated. Could you remove it? Thank!

Reply:
Fixed.

Reply:
Thanks!